src/Host.cpp

```diff
@@ -523,34 +523,6 @@ void Host::sendRaw( QString command )
     mTelnet.sendData( command );
 }
 
-
-/*QStringList Host::getBufferTable( int from, int to )
```

Comment:
Buffers seem to be "off-screen" (invisible) consoles but it is not clear whether these are commented out because they do not work or because what they were designed for is no longer needed!  OTOH they were disabled by Heiko in https://github.com/Mudlet/Mudlet/commit/4c19388733f2dee18669215a5c359589e5f575ab back in 2009-02-11 18:16:08 - that commit has the comment "Feature: Trigger Chains + special icons | misc improvements"...  but it is not clear why disabling this bit is a "misc improvement".... :confused: 

There ARE other commands that work on "Buffers" but it may be that commands that work on other things like consoles or windows now also work on buffers so that these commands with "Buffer" in them are rendered redundant by other less specific commands... :unamused: